### DISCUSSION TITLE
Clarification about the commercial usage

### DISCUSSION THREAD

**Discussion by abudhakeer** [ueberdosis/tiptap#5142] [General Questions] [answered]
Hi there,
Thank you for creating such an awesome tool. Just started to play with yesterday and am already loving it :)

BTW I wanna know the licensing or limitations associated in using this editor with my product which allows to take notes. Basically the product has paid subscriptions or LTD and I wanna make sure that whether it's fine to go ahead using it. 

In [one of the posts](https://news.ycombinator.com/item?id=36960818) I saw it as allowed to be used but then I saw the pricing page and was confused a bit about the usage.

**Comment by nperez0111** [comment, marked as answer]
Hey @abudhakeer, so our tiptap editor is completely [unmodified MIT](https://github.com/ueberdosis/tiptap/blob/main/LICENSE.md) which allows commercial use, we have no intention on ever going back on this. Many products already rely on us. 

The pricing page that you referring to is talking about our pro-extensions and collaboration servers that you can pay for to save you time in developing your product. It acts as a boost to your product, not as a roadblock. All of the pro-extensions are built directly on top of tiptap & hocuspocus (our collaboration server product that is also MIT with a separate proprietary set of tools on top).

I hope this answers your question

**Reply by abudhakeer** [reply]
Hey @nperez0111 
Thank you for your response and this completely answers my question.

I already love how easy it is for me to build my required idea in my product using Tiptap. Gonna launch the updated version soon :)
